CMakeLists.txt

```diff
@@ -19,6 +19,9 @@ if(MSVC)
 else()
 	set(CMAKE_CXX_FLAGS_DEBUG "-g")
 	set(CMAKE_CXX_FLAGS_RELEASE "-O3")
+	set(CMAKE_FIND_LIBRARY_SUFFIXES ".a")
```

Comment:
For this part, do you want to
1. Add a check for Linux explicitly, but leave it in CMakeLists.txt
2. Have it set through a different `if` in the same file
3. Set it through the build command in the yml file

Reply:
I added it, you need to manually authorize the pipeline